pimpmycause/pimpmycause/urls.py

```diff
@@ -38,5 +38,7 @@
     url(r'^accounts/', include('registration.backends.default.urls')),
     url(r'^s3direct/', include('s3direct.urls')),
     url(r'^accounts/profile/', profile_update, name='profile_update'),
-
+    url(r'^user/password/reset/$', 'django.contrib.auth.views.password_reset',
```

Comment:
As mentioned in the ticket there is no need to include any urls, the password reset urls are part of the default django auth backend and are included automatically since they are all imported above. The reset workflow